packages/grafana-schema/src/schema/graph.cue

```diff
@@ -47,6 +51,7 @@ AxisConfig: {
 	axisWidth?:         number
 	axisSoftMin?:       number
 	axisSoftMax?:       number
+	axisGridShow?:      bool | *null
```

Comment:
We've managed to avoid explicit nulls being allowed so far, for <implicit reference to the thousands of words already written> - why break the seal and add it here? The field is already optional